src/js/modules/articles/selectors.js

```diff
@@ -32,7 +32,9 @@ export const getArticlesWithContributors = createSelector(
       if (targetArticle.contributors === undefined) {
         targetArticle.contributors = [];
       }
-      targetArticle.contributors.push(users[authorship.contributorId]);
+      if (!targetArticle.contributors.includes(users[authorship.userId])) {
```

Comment:
Why are you checking? Are there redundant authorships?

Reply:
No, just some re-select repeating stuff.

Reply:
Expand? This is basically checking if the user is already in the contributors list, but that would only happen if there's more than one authorship that connects the two

Reply:
For some reason, the targetArticle already has contributors in it when I call the selector a second time even though it's only supposed to have the raw original article.

Reply:
Is this a recent thing? Could be Redux persist

Reply:
Nah, it happened after I fixed the thing with the deep clone.